src/options.jl

```diff
@@ -125,7 +135,9 @@ function validate_solver_options_and_add_default_values!(options::Options)::Opti
 
         # define type/domain constraints for each known key
         domain_constraints = (v  ->  true)
-        if key == :mode
+        if key == :verbosity
+            expected_type = Bool
```

Comment:
this return type doesn't correspond to [your comment here](https://github.com/JuliaReach/Reachability.jl/issues/7#issuecomment-342611844)? am i missing something?

OTOH this does work ok for me:

```julia
julia> solve(s, :T=>1.0, :verbosity=>0);

julia> solve(s, :T=>1.0, :verbosity=>1);
[info] Time discretization...
[info] elapsed time: 0.000288061 seconds
[info] Reachable States Computation...
[info] elapsed time: 0.012594997 seconds
[info] Projection...
[info] elapsed time: 0.001423731 seconds

julia> solve(s, :T=>1.0, :verbosity=>2);
[info] Time discretization...
[info] elapsed time: 0.000140385 seconds
[info] Reachable States Computation...
[info] elapsed time: 0.007518403 seconds
[info] Projection...
[info] elapsed time: 0.0013399 seconds
```

Reply:
but `true/false` doesn't work

Reply:
Great finding, this was old and it was never tested.